lib/Settings/Admin.php

```diff
@@ -37,6 +39,9 @@ public function getForm(): TemplateResponse {
 		$freePromptPickerEnabled = $this->config->getAppValue(Application::APP_ID, 'free_prompt_picker_enabled', '1') === '1';
 		$speechToTextAvailable = $this->speechToTextManager->hasProviders();
 		$speechToTextEnabled = $this->config->getAppValue(Application::APP_ID, 'speech_to_text_picker_enabled', '1') === '1';
+		$chattyLLMUserInstructions = $this->config->getAppValue(Application::APP_ID, 'chat_user_instructions', $this->l10n->t(Application::CHAT_USER_INSTRUCTIONS));
+		$chattyLLMUserInstructionsTitle = $this->config->getAppValue(Application::APP_ID, 'chat_user_instructions_title', $this->l10n->t(Application::CHAT_USER_INSTRUCTIONS_TITLE));
```

Comment:
Shall we just leave the instructions in English and add an instruction to take over the language of the user?

Reply:
weird since the string should be put in place by php and shouldn't have to do anything with the function call. By "source parser" you mean it is all raw code parsing to get the list of all translatable strings?

Reply:
Yeah, it's using regular expressions ;)

Reply:
:grimacing:

Reply:
> Shall we just leave the instructions in English and add an instruction to take over the language of the user?

Yeah that could be done but different models behave differently with this. I tried asking something in German when the user instruction was in English, it responded in German but explained it in English as well.

But I still think your approach is the way to go. Admins can change the instructions to their liking afterwards.

Reply:
Please remove the ->t calls.